.changelog/epilogue.md

```diff
@@ -0,0 +1,618 @@
+# CHANGELOG
+
+## v0.19.0
+
+*August 22nd, 2022*
+
+#### BREAKING CHANGES
+
+- Remove `height` attribute from `IbcEvent` and its variants
+  ([#2542](https://github.com/informalsystems/ibc-rs/issues/2542))
```

Comment:
We should change all the links `https://github.com/informalsystems/ibc-rs/*` -> `https://github.com/informalsystems/hermes/*`

Reply:
The links are automatically redirected. I thought it would be nice to keep them that way as the links are not seen in the rendered changelog, but happy to change if you think hermes links are better.